Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -194,18 +194,20 @@ function vtkWidgetManager(publicAPI, model) {
 
   publicAPI.enablePicking = () => {
     model.pickingEnabled = true;
+    model.pickingAvailable = true;
+    publicAPI.renderWidgets();
```

Comment:
how about not rendering widgets if pickingEnabled and pickingAvailable were already true ?

Reply:
This was to preserve original functionality, where calling `enablePicking` would render the picking and front buffers regardless of `pickingEnabled`. Since some code already uses this, I decided to keep it this way instead of making it a breaking change.